stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1106,6 +1142,8 @@ function(add_swift_target_library name)
                         "${SWIFTLIB_multiple_parameter_options}"
                         ${ARGN})
   set(SWIFTLIB_SOURCES ${SWIFTLIB_UNPARSED_ARGUMENTS})
+  # FIXME: This is just for source-code backward compatibility
+  set(sdk ${SWIFTLIB_SDK})
```

Comment:
I'm not sure I follow.  What is this maintaining compatibility with?  I think that with this change, we should just update the usage and effectively treat this as an internal implementation that shouldn't be used externally.

Reply:
👍